packages/backend/src/peripherals/ethereum/multicall/MulticallConfig.ts

```diff
@@ -53,6 +49,16 @@ export const OPTIMISM_MULTICALL_CONFIG: MulticallConfigEntry[] = [
   },
 ]
 
+export const BASE_MULTICALL_CONFIG: MulticallConfigEntry[] = [
+  {
+    sinceBlock: 5022,
+    batchSize: 150,
+    address: EthereumAddress('0xcA11bde05977b3631167028862bE2a173976CA11'),
+    encodeBatch: encodeMulticallV2,
+    decodeBatch: decodeMulticallV2,
```

Comment:
Btw since this is multicall3 we could add support for it, but not needed in this PR